### PR TITLE
[TW-1664] flows pinning variables

### PR DESCRIPTION
**Pull request by petewitcher-postman**
- Page is now publicly visible
- Added diagrams and additional context

[View the beta docs](https://tw-1664-flows-pinning-variables.learning.postman-beta.com/docs/postman-flows/concepts/pinning-variables/)

### CODE REVIEW

File: src/pages/docs/postman-flows/concepts/pinning-variables.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Pinning variables in Postman Flows"
```
Comment (mark-anthony-postman) [inline src/pages/docs/postman-flows/concepts/pinning-variables.md:2]:
```suggestion
title: "Pin variables in Postman Flows"
```

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

**Review by rabbah — APPROVED** [review]
Lgtm

**Review by jkonrath-postman — APPROVED** [review]

**Comment by rabbah** [comment]
This is looks good - thanks!
One nit/suggestion: can we add links to 2 flows for the 2 diagrams?

**Review by jkonrath-postman — APPROVED** [review]
LGTM - I like the diagrams!

**Pull request merged**